ui/src/styles.css

```diff
@@ -39,7 +39,7 @@ a {
 }
 
 body {
-  background-color: #F1F1F1;
+  background-color: #FAFBFC;
```

Comment:
nit: it might help to have names for the colors used in the pallet, for easier coordination with UI/UX designers.

Reply:
I like the idea. I think when we convert this file to React we should put this backgroundGrey in `colors.ts`